docs/developer-guide.md

```diff
@@ -25,4 +25,49 @@ This command brings up an [inquirer.js](https://github.com/SBoudrias/Inquirer.js
 
 The general recommendation is to run this changeset command after a feature or bug fix is completed and before creating a pull request. 
 
-A GitHub bot [changeset-bot](https://github.com/apps/changeset-bot) has been enabled that adds a comment to pull requests with changeset information from the branch and includes a warning when no changesets are found. If a changeset was found, a new [GitHub release](https://github.com/SAP/guided-answers-extension/releases) will be created.
\ No newline at end of file
+A GitHub bot [changeset-bot](https://github.com/apps/changeset-bot) has been enabled that adds a comment to pull requests with changeset information from the branch and includes a warning when no changesets are found. If a changeset was found, a new [GitHub release](https://github.com/SAP/guided-answers-extension/releases) will be created.
+
+## Git Guidelines
+
+We adhere to the [Conventional Commits](https://conventionalcommits.org) specification.
+
```

Comment:
Not sure if we need to repeat the information that is already written at https://conventionalcommits.org, but if you want to add some highlights or summary here, I would also include this info:

>1. **fix**: a commit of the type fix patches a bug in your codebase (this correlates with [PATCH](http://semver.org/#summary) in Semantic Versioning).
>1. **feat**: a commit of the type feat introduces a new feature to the codebase (this correlates with [MINOR](http://semver.org/#summary) in Semantic Versioning).
>1. **BREAKING CHANGE**: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with [MAJOR](http://semver.org/#summary) in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.fix: a commit of the type fix patches a bug in your codebase (this correlates with [PATCH](http://semver.org/#summary) in Semantic Versioning).
feat: a commit of the type feat introduces a new feature to the codebase (this correlates with [MINOR](http://semver.org/#summary) in Semantic Versioning).
BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with [MAJOR](http://semver.org/#summary) in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.